frameworks/kafka/src/main/dist/server.properties.mustache

```diff
@@ -27,9 +27,28 @@ broker.id={{POD_INSTANCE_INDEX}}
 #     listeners = security_protocol://host_name:port
 #   EXAMPLE:
 #     listeners = PLAINTEXT://your.host.name:9092
+{{^KAFKA_ENABLE_TLS}}
 listeners=PLAINTEXT://:{{KAFKA_BROKER_PORT}}
+{{/KAFKA_ENABLE_TLS}}
 
-# Hostname and port the broker will advertise to producers and consumers. If not set, 
+{{#KAFKA_ENABLE_TLS}}
+############################# TLS Settings #############################
+listeners=PLAINTEXT://:{{KAFKA_BROKER_PORT}},SSL://:{{KAFKA_BROKER_PORT_TLS}}
```

Comment:
@mhrabovcin Let's make TLS configurable like this:

- "is it enabled?"
- "leave plaintext enabled?" This can be accomplished using an inverted section:
```
{{#KAFKA_ENABLE_TLS_ALLOW_PERMISSIVE}}
listeners=PLAINTEXT://:{{KAFKA_BROKER_PORT}},SSL://:{{KAFKA_BROKER_PORT_TLS}}
{{/KAFKA_ENABLE_TLS_ALLOW_PERMISSIVE}}
{{^KAFKA_ENABLE_TLS_ALLOW_PERMISSIVE}}
listeners=SSL://:{{KAFKA_BROKER_PORT_TLS}}
{{/KAFKA_ENABLE_TLS_ALLOW_PERMISSIVE}}